component-overview/examples/cards/GroupCard-clickable.jsx

```diff
@@ -1,12 +1,17 @@
-import { GroupCard, GroupCardElement } from '@sb1/ffe-cards-react';
+import {
+    GroupCard,
+    GroupCardElement,
+    GroupCardFooter,
+    GroupCardTitle,
+} from '@sb1/ffe-cards-react';
 import { Heading2, Paragraph } from '@sb1/ffe-core-react';
 
 () => {
     return (
-        <GroupCard shadow={true}>
-            <GroupCardElement noPadding={true}>
-                Øverste kort kan være uten padding hvis man ønsker
-            </GroupCardElement>
+        <GroupCard>
+            <GroupCardTitle>
```

Comment:
Spør gjerne Sindre om vi burde ha `aria-labelledby` på GroupCard i exemplena våre 
```
<GroupCard aria-labelledby="heading-id">
            <GroupCardTitle>
                <Heading2 lookLike={5} id="heading-id">Tittel på gruppe</Heading2>
            </GroupCardTitle>
            ```
            
            Jag er ikke sikker på svaret her.. kanskje heading holder.

Reply:
Trenger ikke, sier sindre 👍 Siden det er mulig med headings, så er det nok for å navigere innholdet